src/fabian_nemazi_ex/pa02/test_pa02.py

```diff
@@ -0,0 +1,145 @@
+__author__ = 'Fabian Nemazi', 'Ahmed salaad Ahmed.'
+__email__ = 'Salmaaah@nmbu.no', 'fabinema@nmbu.no'
+
+import chutes_simulation as cs
+import pytest
+
+class TestBoard:
+    """
+    test for board class
+    """
+    def test_constructor(self):
+        """
+        making sure you cant give invalid ladders and chutes input
```

Comment:
The description doesn't really relate to what you did.
If you can't give invalid ladders / snakes, you have to check if it raise a value error when you give invalid arguments.